src/data/accessibility-quiz.ts

```diff
@@ -248,8 +248,17 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA"
   },
   {
-    Question:
-      "How can you make focus elements more accessible to screen readers on native mobile apps?",
+    Question: "What is the best way to make your website keyboard accessible?",
```

Comment:
I would suggest changing this question so it better matches the answer and explanation you provided 

```suggestion
    Question: "What should you avoid doing with the tabindex attribute when it comes to keyboard accessibility?",
```